plugins/dashboard_pi/src/baro_history.cpp

```diff
@@ -284,7 +284,10 @@ void DashboardInstrument_BaroHistory::DrawForeground(wxGCDC* dc)
   m_ratioW = double(m_DrawAreaRect.width) / (BARO_RECORD_COUNT-1);
  // dc->DrawText(wxString::Format(_(" Max %.1f Min %.1f since %02d:%02d  Overall Max %.1f Min %.1f "),m_MaxPress,m_MinPress,hour,min,m_TotalMaxPress,m_TotalMinPress), m_LeftLegend+3+2+degw, m_TopLineHeight-degh+5);
  // Cant get the min sice to work...
- dc->DrawText(wxString::Format(_(" Max %.1f since %02d:%02d  Overall Max %.1f Min %.1f "),m_MaxPress,hour,min,m_TotalMaxPress,m_TotalMinPress), m_LeftLegend+3+2+degw, m_TopLineHeight-degh+5);
+  wxString sMax = _( "Max" ), sSince = _( "since" ), sOverall = _( "Overall" ), sMin = _( "Min" );
```

Comment:
The original code you changed is the way to go and we should be slowly but surely converging to it in all the codebase. Otherwise there will always be
- Questions like "What does this word mean" on Crowdin
- Complaints like "This word is translated wrong, because although it is the same in English, the translation to, say, French differs when talking time and distance"

Bad practice for localization:
`_("Word") + somevariable + _("another word or two") + anothervariable`
Good practice for localization:
`wxString::Format(_("Word %f another word or two %02d"), somevariable, anothervariable)`

Of course it won't resolve every localization glitch out there, but sure is the right direction.